pkg/operator/podsecurityreadinesscontroller/conditions.go

```diff
@@ -59,16 +70,45 @@ func (c *podSecurityOperatorConditions) addViolation(ns *corev1.Namespace) {
 	c.violatingCustomerNamespaces = append(c.violatingCustomerNamespaces, ns.Name)
 }
 
-func makeCondition(conditionType string, namespaces []string) operatorv1.OperatorCondition {
+func (c *podSecurityOperatorConditions) addInconclusive(ns *corev1.Namespace) {
+	if runLevelZeroNamespaces.Has(ns.Name) {
+		c.inconclusiveRunLevelZeroNamespaces = append(c.inconclusiveRunLevelZeroNamespaces, ns.Name)
+		return
+	}
+
+	isOpenShift := strings.HasPrefix(ns.Name, "openshift")
+	if isOpenShift {
+		c.inconclusiveOpenShiftNamespaces = append(c.inconclusiveOpenShiftNamespaces, ns.Name)
+		return
+	}
+
+	if ns.Labels[labelSyncControlLabel] == "false" {
+		// This is the only case in which the controller wouldn't enforce the pod security standards.
+		c.inconclusiveDisabledSyncerNamespaces = append(c.inconclusiveDisabledSyncerNamespaces, ns.Name)
+		return
+	}
+
+	c.inconclusiveCustomerNamespaces = append(c.inconclusiveCustomerNamespaces, ns.Name)
+}
+
+func makeCondition(conditionType, conditionReason string, namespaces []string) operatorv1.OperatorCondition {
+	var messageFormatter string
+
+	if conditionReason == violationReason {
+		messageFormatter = "Violations detected in namespaces: %v"
+	} else if conditionReason == inconclusiveReason {
+		messageFormatter = "Could not evaluate violations for namespaces: %v"
+	}
```

Comment:
What should happen if the condition reason provided is one we don't know about?

When doing the `fmt.Sprintf()` with an empty format string, the message would end up looking something like:
```
%!(EXTRA []string=[foo bar baz])
```

This isn't very helpful to users.

We could either:
- Use a `switch` statement with a `default` case that sets a sane message for any condition reason we don't have specific handling for
- Have the message/message format itself passed as a parameter